elements/lisk-passphrase/test/index.spec.ts

```diff
@@ -12,23 +12,22 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { expect } from 'chai';
 import * as passphrase from '../src';
 
 describe('passphrase index.js', () => {
 	it('should export an object', () => {
-		return expect(passphrase).to.be.an('object');
+		return expect(passphrase).toEqual(expect.any(Object));
```

Comment:
`.toBeObject()`?

Reply:
Could be, but `.toBeObject()` is from `jest-extended`. I prefer to use the basic jest matchers and assertions where it is possible.